apps/allocations/app/components/App/AllocationsHistory.js

```diff
@@ -48,51 +58,51 @@ const AllocationsHistory = ({ allocations }) => {
           recipients.length === 1 ? '1 entity'
             : recipients.length + ' entities',
           description,
-          <Status key={index} code={status} />,
+          <Status key={index} code={2 } />,
           <div
             key={index}
             css={{
               color: theme.negative,
               fontWeight: 600,
             }}>
-            { displayCurrency(BigNumber(amount)) } { token }
+            { displayCurrency(BigNumber(amount)) } { getTokenSymbol(token) }
           </div>
         ]
       }}
-      renderEntryExpansion={({ recipients, amount, token }) => {
-        return recipients.map((recipient, index) => {
-          const allocated = BigNumber(recipient.amount).div(amount)
-          return (
-            <div key={index} css={{
-              display: 'flex',
-              flexDirection: 'column',
-            }}>
-              <IdentityBadge
-                entity={recipient.address}
-                shorten={true}
-              />
-              <div css={{
-                marginTop: '7px',
-                marginBottom: '4px',
-              }}>
-                <ProgressBar
-                  value={allocated}
-                  color={theme.accentEnd}
-                />
-              </div>
-              <div css={{
-                color: theme.contentSecondary,
-                alignSelf: 'flex-end',
-                fontSize: '12px',
-              }}>
-                { displayCurrency(BigNumber(recipient.amount)) } {' '}
-                {token} {' • '}
-                { allocated.times(100).dp(0).toNumber() }{'%'}
-              </div>
-            </div>
-          )
-        })
-      }}
+      //renderEntryExpansion={({ recipients, amount, token }) => {
+      //  return recipients.map((recipient, index) => {
+      //    const allocated = BigNumber(recipient.amount).div(amount)
+      //    return (
+      //      <div key={index} css={{
+      //        display: 'flex',
+      //        flexDirection: 'column',
+      //      }}>
+      //        <IdentityBadge
+      //          entity={recipient.address}
+      //          shorten={true}
+      //        />
+      //        <div css={{
+      //          marginTop: '7px',
+      //          marginBottom: '4px',
+      //        }}>
+      //          <ProgressBar
+      //            value={allocated}
+      //            color={theme.accentEnd}
+      //          />
+      //        </div>
+      //        <div css={{
+      //          color: theme.contentSecondary,
+      //          alignSelf: 'flex-end',
+      //          fontSize: '12px',
+      //        }}>
+      //          { displayCurrency(BigNumber(recipient.amount)) } {' '}
+      //          {token} {' • '}
+      //          { allocated.times(100).dp(0).toNumber() }{'%'}
+      //        </div>
+      //      </div>
+      //    )
+      //  })
+      //}}
```

Comment:
Can we delete this? Is it enough to have it in the commit history?

Reply:
This will be implemented pretty soon in the future, but possibly not for the initial mainnet release. It's not uncommon for A1 to leave pending stuff commented out in their prod code. Not saying we should follow it on that principle, but I think it has it's uses